GAME/GAMEFLOW.C

```diff
@@ -174,17 +179,27 @@ void DoGameflow()//10F5C(<), 10FD8(<)
 
 void LoadGameflow()//102E0, 102B0
 {
+	char* s = NULL;
+#if PC_VERSION
+	void* tmp;
+	LoadFile(GF_SCRIPT_FILENAME, &s);
+#else
 	int len = FILE_Length(GF_SCRIPT_FILENAME);
-	char* s = game_malloc(len);
+	s = game_malloc(len);
+#endif
 	int j = 0;
 	int i = 0;
 	struct STRINGHEADER sh;
 
+	int num_strings, wad_len;
```

Comment:
Looks like you've removed use of STRINGHEADER? Why? Original game uses this, num_strings/wad_len are part of STRINGHEADER